tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -40,17 +41,23 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
         } else {
           line.setMinZoom(10);
         }
+
+        if( disputed.getAsInt() == 1 ) {
+          line.setAttr("disputed", 1);
+        }
       }
     }
   }
 
   @Override
   public List<OsmRelationInfo> preprocessOsmRelation(OsmElement.Relation relation) {
-    if (relation.hasTag("type", "boundary") && relation.hasTag("boundary", "administrative")) {
+    if (relation.hasTag("type", "boundary") && (relation.hasTag("boundary", "administrative") || relation.hasTag("boundary", "disputed"))) {
       Integer adminLevel = Parse.parseIntOrNull(relation.getString("admin_level"));
+      Integer disputed = relation.hasTag("boundary", "disputed") ? 1 : 0;
```

Comment:
This seems to always eval to `0` in error

Reply:
The Line of Control in Cyprus ([r/13562322](https://www.openstreetmap.org/relation/13562322))is an odd ball – it's `admin_level` is `3` for starters. 

I was testing around Israel as the build there has a smaller bounding box than India or China and more `admin_level` = `2` features.

- Israel - Egypt boundary way with relevant relations: [w/95211614](https://www.openstreetmap.org/way/95211614)
- Israel - Gaza boundary way with relevant relations: [w/25057102](https://www.openstreetmap.org/way/25057102)

Reply:
It seems to evaluate to 1 on this relation: https://www.openstreetmap.org/relation/1703814